src/Service/Startup.cs

```diff
@@ -359,6 +361,17 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, RuntimeC
             // https://andrewlock.net/understanding-pathbase-in-aspnetcore/#placing-usepathbase-in-the-correct-location
             app.UseCorrelationIdMiddleware();
             app.UsePathRewriteMiddleware();
+
+            // SwaggerUI visualization of the OpenAPI description document is only available
+            // in developer mode in alignment with the restriction placed on ChilliCream's BananaCakePop IDE.
+            if (runtimeConfigProvider.IsDeveloperMode() || env.IsDevelopment())
+            {
+                app.UseSwaggerUI(c =>
+                {
+                    c.SwaggerEndpoint("/openapi", "DataApibuilder-OpenAPI-PREVIEW");
```

Comment:
Should we prepend the rest base bath before `openapi`?

Reply:
I agree in principle. This requires changes to the restcontroller/restservice. I'm evaluating that now.

Reply:
Updated, there is more code in RestService/RestController/Startup and a new helper class SwaggerEndpointMapper which helps achieve this for local, late config, and custom rest route configured environments.